Monika After Story/game/screens.rpy

```diff
@@ -42,30 +38,27 @@ init python in mas_layout:
 
     QUIT_YES = store.layout.QUIT_YES
     QUIT_NO = store.layout.QUIT_NO
-    QUIT = "Leaving without saying goodbye, [player]?"
-    UNSTABLE = (
-        "WARNING: Enabling unstable mode will download updates from the " +
-        "experimental unstable branch. It is HIGHLY recommended to make a " +
-        "backup of your persistents before enabling this mode. Please report " +
-        "issues found here with an [[UNSTABLE] tag."
+    QUIT = _("Leaving without saying goodbye, [player]?")
+    UNSTABLE = _(
+        "WARNING: Enabling unstable mode will download updates from the experimental unstable branch. It is HIGHLY recommended to make a backup of your persistents before enabling this mode. Please report issues found here with an [[UNSTABLE] tag."
```

Comment:
long string lines are split to keep the character line limit to 80-120 characters. please split these lines like they were before.